client/README.md

```diff
@@ -11,20 +11,24 @@ Two current options are to [access Matrix and contact a bot](https://wiki.polkad
 ## Development
 
 To develop you need two env variables:
-- `VITE_CAPTCHA_KEY`: The [reCaptcha v3 site key](https://www.google.com/u/1/recaptcha/admin).
-- `VITE_FAUCET_URL`: The endpoint to contact the faucet.
+- `PUBLIC_CAPTCHA_KEY`: The [reCaptcha v2 site key](https://www.google.com/u/0/recaptcha/admin).
+- `PUBLIC_FAUCET_URL`: The endpoint to contact the faucet.
 
-The reason for which these variables have `VITE` as a prefix is a security meassure to not upload any unnecesary data. [More info here](https://vitejs.dev/guide/env-and-mode.html#env-files)
+The reason for which these variables have `PUBLIC_` as a prefix is a security meassure to not upload any unnecesary data. [More info here](PUBLIC)
```

Comment:
`[More info here](PUBLIC)`

Forgot to fill in the url here?

Reply:
Oh right, I thought I pasted it, good catch.